FlowCrypt/Functionality/DataManager/Encrypted Storage/EncryptedStorage.swift

```diff
@@ -14,16 +14,14 @@ import UIKit
 protocol EncryptedStorageType {
     var storage: Realm { get }
 
+    var activeUser: User? { get }
     func getAllUsers() -> [User]
     func saveActiveUser(with user: User)
-    var activeUser: User? { get }
-    func doesAnyKeyExist(for email: String) -> Bool
+    func doesAnyKeypairExist(for email: String) -> Bool
 
     func addKeys(keyDetails: [KeyDetails], passPhrase: String?, source: KeySource, for email: String)
     func updateKeys(keyDetails: [KeyDetails], passPhrase: String?, source: KeySource, for email: String)
```

Comment:
Let's call it `addKeypairs` and `updateKeypairs`, but really, is there any difference between them? Whether I'm adding or updating, I want the right keypair to be there in the end. I would refactor these two as a single method: `putKeypairs`, which adds or updates each keypair depending if we already have it or not (for that user email).

Reply:
Yes, logic looks the same except that `addKeys` has `UpdatePolicy.error`, and `updateKeys` has `UpdatePolicy.all` update policy. We could leave only `updateKeys` if we are find with such policy.

Reply:
Let's leave only `updateKeys` but call it `upsertKeypairs`

Reply:
Let's stay on `putKeypairs` if you don't mind.